core/jobs.py

```diff
@@ -489,6 +567,8 @@ def map(item):
           WARNING: The OutputWriter converts mapper output keys to type str.
           So, if you have keys that are of type unicode, you must yield
           "key.encode('utf-8')", rather than "key".
+          Raises:
```

Comment:
No need to document NotImplementedError -- this just indicates that a subclass method should be called instead of this one.